contracts/interfaces/IBalancerVault.sol

```diff
@@ -0,0 +1,54 @@
+pragma solidity ^0.8.0;
```

Comment:
is this a full balancer interface? lets have it full so we won't return to it

Reply:
no, it is not. I would be just importing it from Balancer's repo but they have it versioned for 0.7.x so need to copy over the interface. As of now, it is just the sections I was using but will make it full-fledged.